uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/filters/date-type.js

```diff
@@ -1,38 +1,35 @@
 define([
-    'tasks-assignments/filters/filters'
-], function (filters) {
+    'tasks-assignments/filters/filters',
+    'common/lodash'
+], function (filters, _) {
     'use strict';
 
     filters.filter('filterByDateType',['$filter', '$rootScope', '$log', function ($filter, $rootScope, $log) {
         $log.debug('Filter: filterBy.dateType');
 
-        return function(inputArr, dateTypeArr) {
-            var filteredArr = [],
-                i = 0,
-                ii,
-                inputArrLen = inputArr.length,
-                dateTypeArrLen = dateTypeArr.length,
-                dateContactList,
-                dateContactListLen;
-
-            if (!inputArrLen || !dateTypeArrLen) {
-                return inputArr;
+        return function(dateContactList, dateTypeList) {
```

Comment:
I think we need a description here on the function. At the moment we need to look inside the func to understand what `dateContactList` and `dateTypeList` are and what kind of data it returns.